api/templates/gen/openapi.yaml

```diff
@@ -0,0 +1,70 @@
+swagger: "2.0"
+info:
+  title: Templates Service
+  description: A service for applying templates to Keboola projects
+  version: "1.0"
+host: templates.keboola.com
+consumes:
+- application/json
+- application/xml
+- application/gob
+produces:
+- application/json
+- application/xml
+- application/gob
+paths:
+  /:
+    get:
+      tags:
+      - templates
+      summary: index-root templates
+      operationId: templates#index-root
+      responses:
+        "307":
```

Comment:
Asi si to zabudol pregenerovat (?):
![image](https://user-images.githubusercontent.com/19371734/156160213-aede87b3-8a95-41b5-ab01-1b56b3b072e2.png)

Reply:
Jo 👍